v0/destinations/rockerbox/transform.js

```diff
@@ -6,7 +6,8 @@ const {
   constructPayload,
   getSuccessRespEvents,
   getErrorRespEvents,
-  getHashFromArray
+  getHashFromArray,
```

Comment:
Is `getDestinationExternalID` being used ?

Reply:
Removing this as it is unused now.